build_sherlock_platform.sh

```diff
@@ -17,4 +17,7 @@ BUILD_PROPERTIES=(
   "-Dintellij.build.incremental.compilation.fallback.rebuild=false"
 )
 
+# Calls getPlugins.sh to update/download android repo before building
```

Comment:
If it doesn't take too long to clone the jetbrains/Android repo, LGTM. But, if it takes some time, I wonder if we should consider making this optional? E.g., `--with-android`?

This repo was required when I was running from inside IntelliJ IDEA. It was not required when running `build_sherlock_platform.sh`, if I'm not mistaken.

Reply:
That's correct. I haven't faced any errors without it. 
Although it didn't take time for my (powerful) machine to clone the repo, I like the idea of having a flag for it. Updated it accordingly.